src/react-apps/applications/dashboard/__tests__/components/base/knowIssues.test.tsx

```diff
@@ -0,0 +1,57 @@
+
+  });
+
+  it('+++ should handle updating knownIssues state with stripped version of knowIssues page ftom github', () => {
+    const mountedComponent = mount(
+      (
+        <KnownIssuesComponent
+          language={mockLanguage}
+          classes={mockClasses}
+        />
+      ),
+    );
+
+    const instance = mountedComponent.instance() as KnownIssuesComponent;
+    const getSpy = jest.spyOn(networking, 'get').mockImplementation(() => Promise.resolve(mockGetResult));
+    instance.componentDidMount();
+    expect(instance._isMounted).toBe(true);
+    return Promise.resolve().then(() => {
```

Comment:
If you use "async/await" - await Promise.resolve();
i think you get less, and easier, to read code.